docs/habitat.md

```diff
@@ -151,6 +123,38 @@ $ habitat profile create ~/profiles/frontend1
 [2017-03-28T13:30:18-04:00] INFO: Copying artifact to /Users/aleff...
 ```
 
+### inspec habitat profile setup
+
+Create a Habitat directory that includes a plan file, config hooks, and more in a profile directory.
+
+This is the same process that is used by `inspec habitat profile create` - but this adds the generated Habitat
+ directory and file to your system so that you can commit them to source control. If you commit these files to Github, you can connect that plan to the [https://www.habitat.sh/docs/using-builder/](Habitat Builder Service).
```

Comment:
I think you have that hyperlink syntax backwards - it's:
```
[Link Text](url)
```
[Like This](https://docs.chef.io)

It misrenders:
![screen shot 2018-10-31 at 20 33 47](https://user-images.githubusercontent.com/156460/47826164-7d0c2100-dd4c-11e8-8336-b357a2cb25d4.png)

Reply:
Great catch! A heuristic I use is "squared circle". Not sure if it helps though.